template/Makefile.in

```diff
@@ -479,6 +479,9 @@ _PREFIXED_SYMBOL = TOKEN_PASTE($(SYMBOL_PREFIX),name)
 	@$(ECHO) making dummy $(DEST_FILE)
 	$(Q)echo '#include "$(*F).dmyh"' > $@
 
+rjit_funcs.c: rjit_funcs.c.template yjit/bindgen/src/main.rs
+	$(Q) grep "allowlist_function" $(word 2,$^) | grep -v '|' | sed 's/^[^"]*"\([^"]*\)".*$$/CHECK(\1)/' | sed '/FUNCTION_START\;/r /dev/stdin' $< > $@
```

Comment:
This seems depending on GNU make.and existence of `/dev/stdin`.

Reply:
I'm not Makefile expert 😅

What part is GNU specific? Also, are there platforms without /dev/stdin?

If I change to use a Ruby script to generate this, would that be more portable?

Reply:
> What part is GNU specific? Also, are there platforms without /dev/stdin?

IIRC, GNU make and some others provide `$(word)` and `$^`, but not all.
Not sure about platforms without `/dev/std*` nowadays.

> If I change to use a Ruby script to generate this, would that be more portable?

By using miniruby?

Reply:
> IIRC, GNU make and some others provide `$(word)` and `$^`, but not all.

Ok, do you have any recommendations to fix this?  I wanted to avoid hardcoding `yjit/bindgen/src/main.rs`, but maybe it's ok to do.

> Not sure about platforms without `/dev/std*` nowadays.

I searched around, and I guess maybe some AIX systems? I'll try to find a solution that doesn't depend on `/dev/std*`. I guess using a tempfile would work (in worst case).